packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 49.10.0 - September 16, 2020
+
+## Added
+
+#### salesforcedx-vscode-soql
+
+- SOQL SObject Integration ([PR #2488](https://github.com/forcedotcom/salesforcedx-vscode/pull/2488))
```

Comment:
This piece and the header above it still needs to be removed. The rest looks good to me.